.github/workflows/shared-build-deploy-container.yml

```diff
@@ -115,7 +115,7 @@ jobs:
 
   validate-event:
     name: "Block manual releases to staging and production"
-    needs: [ pick-environment ]
+    needs: [ pick-environment, pick-runner ]
```

Comment:
This will improve the flow as below:
<img width="1297" alt="image" src="https://github.com/trakx/github-actions/assets/6867927/f3a9ca5b-ab60-4814-8e62-d0f440d53408">